### DISCUSSION TITLE
Interpretation of Residual Phase RMS plot

### DISCUSSION THREAD

**Discussion by mosyhey** [insarlab/MintPy#889] [Q&A] [answered]
> **Residual RMS for noise evaluation,** is the phase component that can neither be corrected nor be modeled as ground deformation, thus, is used to characterize the noise level of the InSAR time series. (Yunjun et al., 2019)

Does this plot show deformation noise too? How should it be interpreted?

A study in a seismic focal zone showed that the RMS value reaches the maximum just before an earthquake, while after that, it quickly decreases and reaches the minimum.
![phase_rms_3](https://user-images.githubusercontent.com/105506772/200110916-9fe6c917-8503-4102-939d-42fdfbacc32e.png)
Could it be a type of earthquake precursor that shows a deformation anomaly or a complex deformation? What is your opinion?

**Comment by yunjunz** [comment, marked as answer]
You may want to include a step function during the `correct_topography` step, if you have not done so. For time-series including earthquakes, this usually helps to reduce the residual phase RMS.

Solely based on the RMS plot, I am not sure of a more physical interpretation beyond a coincidence, for the extra large RMS before the earthquake. I would think the phase spatial pattern tells more.

**Reply by mosyhey** [reply]
Thank you dear developer.
